src/org/dokuwiki/translatorBundle/Tests/Services/Language/LanguageFileParserTest.php

```diff
@@ -337,10 +404,12 @@ function testCompleteFile() {
  * german language file
  *
  * @license    GPL 2 (http://www.gnu.org/licenses/gpl.html)
+ *
+ * @package DokuWiki\lang\de\settings
 ';
 
         $this->assertEquals($expectedheader, $parser->getHeader());
-        $this->assertEquals(18, count($parser->getAuthor()->getAll()));
+        $this->assertEquals(20, count($parser->getAuthor()->getAll()));
```

Comment:
I guess this test doesn't make too much sense as this value will probably change over time? or is it working on a static copy?

Reply:
It parses a static copy, so numbers are directly related to src/org/dokuwiki/translatorBundle/Tests/Services/Language/testLang.php